tests/GithubRetry.py

```diff
@@ -132,8 +132,8 @@ def mock_retry_now(self, now):
         else:
             attr = "github.GithubRetry._GithubRetry__datetime"
         with mock.patch(attr) as dt:
-            dt.now = mock.Mock(return_value=datetime.utcfromtimestamp(now))
-            dt.utcfromtimestamp = datetime.utcfromtimestamp
+            dt.now = mock.Mock(return_value=datetime.fromtimestamp(now, timezone.utc))
```

Comment:
The Mock is not relly needed to patch the method, let's remove it
```suggestion
            dt.now = lambda tz=None: datetime.fromtimestamp(now, tz=tz)
```

Reply:
Thanks!

Reply:
Applied!

Reply:
Removed the unused import too.